### DISCUSSION TITLE
Color per app

### DISCUSSION THREAD

**Discussion by Saicobo** [TrianguloY/URLCheck#308] [Ideas]
It would be great to identify which browser or app you will use.
It happens to me that I use 2 or more identical browsers because I have different accounts in both.
I would like to be able to show the app icon or edit the name that will be displayed for the app in the interaction menu. 🥹👑👍🏼 yes?

**Comment by TrianguloY** [comment]
There have been several suggestions with improvements of the open module in general, like this one: https://github.com/TrianguloY/UrlChecker/issues/47

I think that, after finishing the backup feature, I'll need to address this (I would also like to have multiple icons for faster choosing). 
I just...don't know how much it will take me, sorry 😞 (but as usual, prs are welcomed!).
